api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortQueryBuilder.java

```diff
@@ -23,38 +24,31 @@
 @Service
 public class CohortQueryBuilder {
   private static final String REVIEW_TABLE = "cb_review_all_events";
+  private static final String SEARCH_PERSON_TABLE = "cb_search_person";
+  private static final String PERSON_TABLE = "person";
+
   private static final String COUNT_SQL_TEMPLATE =
       "select count(*) as count\n"
-          + "from `${projectId}.${dataSetId}.${table}` ${table}\n"
+          + "from `${projectId}.${dataSetId}.cb_search_person` cb_search_person\n"
```

Comment:
No reason to dynamically set the table when it was always the same and it improves readability